tests/check_check_sub.c

```diff
@@ -2693,8 +2917,11 @@ void exit_handler ()
 
 START_TEST(test_ignore_exit_handlers)
 {
+  int result;
+
   record_test_name(tcase_name());
-  int result = atexit(exit_handler);
+
+  result = atexit(exit_handler);
```

Comment:
If you did all this by hand, I give you credit for your dedication. (:

Reply:
Yes I did it by hand. I think, there is no other way to do it.
In the end, it turned out as somewhat more work as initially expected ("just replace stdbool.h").
But it should be in a form that can be maintained in the project, so the test has to work as well, right?